docker-compose.local.yml

```diff
@@ -1,33 +1,32 @@
-
 services:
-  postgres: 
+  postgres:
     image: postgres
     container_name: postgress_db
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: password
       POSTGRES_DB: mydb
     ports:
-      - "5432:5432" # Data base port
+      - '5433:5432' # Data base port
```

Comment:
i don't think this local port change should be pushed? maybe it's more convenient for more devs, but standard is 5432